parsers/www.giverecipe.com.js

```diff
@@ -0,0 +1,29 @@
+const { reduce } = require('lodash')
+const { parseIngredient } = require('ingredient-parser')
+const { duration } = require('../src/duration')
+
+exports.description = '.tasty-recipes-description'
+
+exports.yield = $ =>
+  $('.tasty-recipes-yield > span')
+    .first()
+    .text()
+
+exports.duration = duration('.tasty-recipes-total-time')
```

Comment:
I don't think you need to wrap this in `duration`, (or you shouldn't if you do). Since the name `duration` is the same as the export, it _should_ get picked up here: https://github.com/plate0/recipe-parser/blob/master/src/parser.js#L13

Reply:
Ah, you are right! I think this was leftover from when I was using a more complicated Cheerio selector. Removed.